.github/actions/git-tag/action.yml

```diff
@@ -23,13 +23,15 @@ runs:
       run: |
         if [[  ${{ inputs.branch_name }} == "master" ]]; then
           VERSION=$(grep -m1 version package.json | awk '{ print $2 }' | sed 's/[", ]//g')
-        fi;
 
-        echo "git tag -a ${VERSION} -m ${VERSION}"
-        if [[ ${{ inputs.dry-run }} == false ]]; then
-          git tag -a ${VERSION} -m "${VERSION} [ci skip] "
-          git remote rm origin
-          GITHUB_REPO=https://${{ inputs.github_token }}:x-oauth-basic@github.com/Alfresco/alfresco-content-app.git
-          git remote add origin $GITHUB_REPO
-          git push origin --tags
+          echo "git tag -a ${VERSION} -m ${VERSION}"
```

Comment:
As I read it, if run from `workflow_dispatch` VERSION will not be set and that will break

Reply:
so what's the suggestion in this case?

Reply:
since there is really nothing to dry-run in this action (without touching anything), I would move the conditional on the step calling this action and remove those branch_name and dry-run inputs

Reply:
can we have this as a separate review? or this will affect the release on master?